docs/en/operation/User-CLI.md

```diff
@@ -402,7 +453,7 @@ Dumping journal of type EMBEDDED to /Users/alluxio/journal_dump-1602698211916
 2020-10-14 10:56:52,254 INFO  RaftJournalDumper - Read 223 entries from log /Users/alluxio/alluxio/journal/raft/02511d47-d67c-49a3-9011-abb3109a44c1/current/log_0-222.
 ```
 
-> Note: This command does not require the Alluxio cluster to be running.
+> Note: This command may fail for certain UFS types if the Alluxio cluster is running due to file locking.
```

Comment:
For readJournal command, this command requires the Alluxio cluster is not running. Otherwise, the journal could be locked and unreadable.